vpc/Makefile

```diff
@@ -0,0 +1,17 @@
+refresh: creds fetch generate
+
+creds:
+	newt --app-type awscreds refresh -r arn:aws:iam::686589254418:role/titus_us_east_1_agents_test_titus titus_us_east_1_agents_test
```

Comment:
minor: change to `weep file arn:aws:iam::686589254418:role/titus_us_east_1_agents_test_titus -f -p titus_us_east_1_agents_test`